pkg/csi/service/common/commonco/k8sorchestrator/k8sorchestrator.go

```diff
@@ -1050,6 +1050,30 @@ func (c *K8sOrchestrator) IsFSSEnabled(ctx context.Context, featureName string)
 			"Setting the feature state to false", featureName, c.internalFSS.configMapName)
 		return false
 	} else if c.clusterFlavor == cnstypes.CnsClusterFlavorWorkload {
+		// Check if it is WCP defined feature state.
+		if _, exists := common.WCPFeatureStates[featureName]; exists {
```

Comment:
Nit: This if{} block is only needed for podVMOnStretchSupervisor FSS and not for other features. So, you can avoid this if{} block execution by checking the FSS name.

Reply:
In future, we want to use this functionality for other WCP initiated features as well, so keeping it generic.